SeaPublicWebsite/ExternalServices/Models/BreRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace SeaPublicWebsite.ExternalServices.Models
```

Comment:
This file and all the BRE specific enums and the API file would be better off in a specific folder together like `ExternalServices/Bre/`